defaults/main.yml

```diff
@@ -10,19 +10,19 @@ apache_server_name: example.com
 apache_htpasswd:
  - filename: default
    user: user
-   password: "$apr1$o0Fo.P8i$iWHLQ.V5hYJ8Xyhsxq/270"
+   password: "$apr1$v9CvCAag$Ow.1dk44cqh5cd87FYE6M1"
 
 # Default Password = admin
 phpmyadmin_htpasswd:
```

Comment:
Variable name needs to be changed.
For BC, we can map the old variable to the new one by default.